app/views/idv/in_person/state_id/show.html.erb

```diff
@@ -106,7 +106,7 @@
   <div class="margin-bottom-4">
     <%= render ValidatedFieldComponent.new(
```

Comment:
Please add the before action and method to `app/controllers/idv/in_person/state_id_controller.rb` for this view.  You would not have know about this in the flow. This is new state id view. The one you observed in the flow is the FSM version. Same reason as other tickets- we need to update the new (not yet used) version so that when we move to it it is not broken. Reach out for a quick demo on this- will be easier to show.  (Also, I can show you remote flow- I tested it too.)

before_action :set_usps_form_presenter

def set_usps_form_presenter
  @presenter = Idv::InPerson::UspsFormPresenter.new
end